python/pyarrow/orc.py

```diff
@@ -54,9 +54,25 @@ def nrows(self):
         return self.reader.nrows()
 
     @property
-    def nstripes(self):
-        """The number of stripes in the file"""
-        return self.reader.nstripes()
+    def file_version(self):
+        """Format version of the ORC file, must be 0.11 or 0.12"""
+        return self.reader.file_version()
+
+    @property
+    def compression(self):
+        """Compression codec of the file"""
+        return self.reader.compression()
+
+    @property
+    def compression_size(self):
+        """Number of bytes to buffer for the compression codec in the file"""
+        return self.reader.compression_size()
```

Comment:
Does this map to the option "compression_block_size" ? If so, maybe using the same name would help (I know the C++ liborc API is consistent though)

Reply:
Nope.

Reply:
Can you then explain what the number this returns exactly means?

Reply:
Actually I found that they are the same in practice. Need to talk to the ORC community about that.